openedx_external_enrollments/api/v0/views.py

```diff
@@ -7,8 +7,8 @@
 from opaque_keys.edx.keys import CourseKey
 from rest_framework import status
 from rest_framework.views import APIView
-from openedx.core.lib.api.authentication import OAuth2Authentication
 
+from openedx.core.lib.api.authentication import OAuth2Authentication
```

Comment:
This needs a backend abstraction. @ivanvgh

Reply:
Done.